cnd/src/http_api/routes.rs

```diff
@@ -202,7 +204,6 @@ pub async fn handle_get_halight_swap(
     }
 }
 
-// TODO: Refactor with make_finalized_swap_entity
```

Comment:
Instead of describing what someone needs to do, can you shortly explain why this should be refactored in this manner please? 
This might also indeed be better stored in a ticket as otherwise this will get lost here.

Reply:
I Wrote this todo and actioned it. I wonder how it go in dev.